catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/AsyncFileAlterationObserver.java

```diff
@@ -260,15 +305,14 @@ private void commitMatch(AsyncFileEntry entry, boolean success) {
    */
   private void doDelete(AsyncFileEntry entry, final AsyncFileAlterationListener listenerCopy) {
     if (!entry.isDirectory()) {
```

Comment:
Same comments as above. This code was actually written early last year. I'm assuming they knew whether or not to null check this value, but I could be wrong. I think for now it might be out of the scope of this ticket, but if you think it's worth looking into later let me know!